dll/shellext/shellbtrfs/recv.cpp

```diff
@@ -27,6 +27,7 @@
 
 
 #ifndef _MSC_VER
+#define __cpuidex __cpuidex_ // prevent redeclaration
 #include <cpuid.h>
```

Comment:
I wonder what would happen if we were just removing this whole `ifndef _MSC_VER` . Would it still build on updated GCC or break completely?

Reply:
It would require to remove the code block at line 1368 as well. I tred to keep changes minimal here.